### PR TITLE
Raven Sears Bowling Javascript

### PR DESCRIPTION
**Pull request by ravensears**

### CODE REVIEW

File: bowling.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+describe('Bowling', () => {
+  it('initalizes a new game', () => {
+    const bowling = new Bowling();
+    expect(bowling.game()).toEqual(0);
+  });
+
+  it('adds a score', () => {
+    const bowling = new Bowling();
+    bowling.addScore(4);
+    expect(bowling.game()).toEqual(4);
```
Comment (ElliottZaki) [inline bowling.test.js:12]: Good use of descriptions and the tests look excellent. Easy to follow.

File: bowling.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  it('recognises a strike', () => {
+    const bowling = new Bowling();
+    bowling.addScore(10);
+    expect(bowling.game()).toBe(10);
+  });
+
+  it('recognises a gutter game', () => {
+    const bowling = new Bowling();
+    for (let i = 0; i > 10; i++) { bowling.addScore(0); }
```
Comment (ElliottZaki) [inline bowling.test.js:23]: Awesome use of a loop!

**Review by ElliottZaki — COMMENTED** [review]
This is a really good start! A bit more tinkering and id say it'll almost be complete! Your tests and descriptions are easy to follow. 10/10 so far! Well done!!

**Comment by ravensears** [comment]
Thanks Elliot. Really appreciate the feedback!

**Pull request closed without merging**